GuardianConnect/Classes/API/GRDHousekeepingAPI.m

```diff
@@ -16,7 +16,9 @@ - (NSMutableURLRequest *)requestWithEndpoint:(NSString *)apiEndpoint andPostRequ
     NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:URL];
     
     [request setValue:[[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleVersion"] forHTTPHeaderField:@"X-Guardian-Build"];
-    
+    if ([[GRDVPNHelper sharedInstance] connectAPIKey]){
+        [request setValue:[[GRDVPNHelper sharedInstance] connectAPIKey] forHTTPHeaderField:@"x-api-key"];
```

Comment:
`x-` is not a good prefix to actively discouraged from being used since they cause all kinds of other trouble down the road so it is better to avoid it to begin with https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

This needs to be changed to `GRD-` everywhere please

Reply:
`GRD-API-Key`